src/plugins/ini/ini.c

```diff
@@ -84,6 +84,8 @@ static int iniKeyToElektraKey (void *vhandle, const char *section, const char *n
 
 	keyAddBaseName (appendKey, name);
 
+	if(value == NULL)
+		keySetMeta(appendKey, "ini/empty", "");
```

Comment:
tom-wa wrote:
> i couldn't really differentiate between 
> `key = ` and `key` without the metakey when writing back to the config file

Ok, I see. So is the following correct?

- null keys are section (status quo)
- `key` is an empty string with ini/empty set
- `key = ` is an empty string

Please correct my statements and add them to README.md.
You also need to rebase, I modified the README.md.